src/ucp/wireup/wireup.c

```diff
@@ -190,8 +191,15 @@ static void ucp_wireup_process_request(ucp_worker_h worker, const ucp_wireup_msg
         }
     }
 
+    /* Use default parameters to initialize local lanes.
+     * This might not always work - need to pass some parameters in the wireup
```

Comment:
Added in TODO here: https://github.com/openucx/ucx/pull/1461/files#diff-8fd5f37669b4533d1e9389ff6839dcccR210